$NetBSD$

Fix compiled-in default path to sendmail; it should be /usr/sbin/sendmail
on essentially all even halfway modern systems.

--- src/slrnfeat.hin~	2008-02-09 18:32:17.000000000 +0000
+++ src/slrnfeat.hin
@@ -179,7 +179,7 @@
 #   define SLRN_SIGNATURE_FILE		".signature"
   /* Changing these here has no effect when using ./configure */
 #   ifndef SLRN_SENDMAIL_COMMAND
-#    define SLRN_SENDMAIL_COMMAND	"/usr/lib/sendmail -oi -t -oem -odb"
+#    define SLRN_SENDMAIL_COMMAND	"/usr/sbin/sendmail -oi -t -oem -odb"
 #   endif
 #  endif			       /* unix */
 # endif				       /* os2 */
